core/shape-parts.cpp

```diff
@@ -39,6 +39,32 @@ const Path& BasicBezier::generate(Point endPoint, Point transitionPoint) {
   return m_path;
 }
 
+const Path& FlatBezier::generate(Point endPoint, Point transitionPoint) {
+  MCFixed flatEndY = m_flatWidth / -2; // WHY DO I HAVE TO MAKE THIS NEGATIVE TO GET IT TO WORK???
```

Comment:
Thanks Mike, That matches with what I thought. The problem I'm having is that if I remove the negative sign then I end up with a tail like this (I also have to comment out the assert in shapes.cpp line 481 to see what's going on):
![bowtietail](https://cloud.githubusercontent.com/assets/17330572/17826007/bc8570fc-66c0-11e6-8e99-ab17aeb40fac.JPG)
Somehow the sign of this end of the bezier curve seems to get reversed somewhere along the line. I'll keep digging as the negative I out in is clearly a work around that's not great!